plaso/parsers/sqlite_plugins/chrome_history.py

```diff
@@ -15,8 +15,14 @@ class ChromeHistoryFileDownloadedEventData(events.EventData):
   """Chrome History file downloaded event data.
 
   Attributes:
+    danger_type (int): enum indicating if the download was dangerous.
+    end_time (int): when the download ended.
```

Comment:
Why not make this a separate event? Also what does the int represent?

Reply:
Updated description. I don't feel it's that useful as its own event, but rather useful context for the download event as a whole, so adding as an attribute seemed appropriate.

Reply:
this deviates from Plaso's current approach, especially if for later usage in automated analysis.

Reply:
happy to provide more detail in a chat if that helps to clarify the current approach

Reply:
I assume you wish to track the association (https://github.com/log2timeline/plaso/issues/3566) between the 2 events and represent this as part of the output?